### ISSUE TITLE
MDTabStripItem only support two Element on Image and Label

### ISSUE THREAD

**Issue by alexonozor** [nativescript-community/ui-material-components#447]
MDTabStripItem  only support 2 element in it's content i.e Image and Label. When we try to add the 3rd Label or element it's all cleared up every content. It will be nice if we can add more content to the tab for like indication or something

**Comment by farfromrefug** [comment]
@alexonozor I dont think it is possible. if you want more you need to create a custom tabstrip
